pkg/services/store/entity/sqlstash/sql_storage_server_test.go

```diff
@@ -123,6 +126,75 @@ func TestCreate(t *testing.T) {
 	}
 }
 
+func TestList(t *testing.T) {
+	s := setUpTestServer(t)
+
+	tests := []struct {
+		name string
+		key  string
+
+		errIsExpected bool
+	}{
+		{
+			"request with invalid key lister",
+			"/playlist.grafana.app",
+			true,
+		},
+		{
+			"request with key lister",
+			"/playlist.grafana.app/playlists",
+			false,
+		},
+		{
+			"request with namespaced key lister",
+			"/playlist.grafana.app/playlists/default",
+			false,
+		},
+		{
+			"request with specific item key lister",
```

Comment:
We could also includes a subresource here to ensure that the test case of having a subresource is also included.  Or it could be included as an additional test case.